Card/Creatable/Safe.ts

```diff
@@ -0,0 +1,38 @@
+import * as isoly from "isoly"
+import * as authly from "authly"
+import { Expires } from "../Expires"
+import { Scheme } from "../Scheme"
+
+export interface Safe {
+	iin: string
+	last4: string
+	scheme: Scheme
+	token: authly.Token
+	expires: Expires
+	csc?: "matched" | "mismatched" | "present"
+	country?: isoly.CountryCode.Alpha2
+	verification?: { type: "pares" | "method" | "challenge" | "guard"; data?: string | { [property: string]: any } }
+}
+
+export namespace Safe {
+	export function is(value: Safe | any): value is Safe {
+		return (
+			typeof value == "object" &&
+			value != null &&
+			typeof value.iin == "string" &&
+			typeof value.last4 == "string" &&
```

Comment:
Length checks, at least on last four, and the other one 6 or 6-8 characters long (don't know if we are allowed to store 8 there?)
```suggestion
			typeof value.iin == "string" &&
			value.iin.length == 6 &&
			typeof value.last4 == "string" &&
			value.last4.length == 4 &&
```